django/river/settings/base.py

```diff
@@ -260,6 +260,7 @@
 OIDC_STORE_ACCESS_TOKEN = True
 # Silently re-authenticated after following time:
 OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS = int(os.environ.get("OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS", 12 * 60 * 60))
+OIDC_TOKEN_USE_BASIC_AUTH = os.environ.get("OIDC_TOKEN_USE_BASIC_AUTH", False) == "True"
```

Comment:
what's this ?

Reply:
The hydra clients (i'm testing with local-cohort-client) we use have this property    "token_endpoint_auth_method": "client_secret_basic",

Reply:
thanks

Reply:
Can you add a comment to remember why it's use